components/grid/grid.css

```diff
@@ -0,0 +1,50 @@
+body {
+  font-family: var(--nunito);
+  display: grid;
+  justify-items: center;
+}
+
+.showcase,
+.showcase-fix {
+  display: grid;
+  border: 0.1rem solid var(--gray-lighter);
+  padding: var(--spacing-l);
+  margin: var(--spacing-l);
+  width: 40rem;
+}
+
+.showcase-fix {
+  height: 10rem;
+}
+
+.col {
+  display: grid;
+  align-items: center;
+  padding-left: var(--spacing-sm);
+  height: 2.5rem;
+  border: 1px solid var(--gray-dark);
+}
+
+.grid-col2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-self: end;
+}
+
+.grid-col3 {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.grid-col4 {
+  display: grid;
```

Comment:
Instead of repeating grid property. You can move grid to utility file and use the classname.

Reply:
I did not want to add too many classnames in my html code thats why i'hv added these properties in my css files where i'm grabbing the element anyways.